tests/test_attributes.py

```diff
@@ -190,6 +190,41 @@ class Uuid(dj.Manual):
     yield Uuid
     Uuid.drop()
 
+@pytest.fixture
+def ParentPart(schema):
+    @schema
+    class ScanData(dj.Manual):
+        definition = """
+        scan_id : int unsigned
+        ---
+        data: int unsigned
+        """
+        
+    @schema
+    class ProcessScanData(dj.Computed):
+        definition = """
+        -> ScanData # Forigen Key Reference
+        ---
+        processed_scan_data : int unsigned
+        """
+            
+        class ProcessScanDataPart(dj.Part):
+            definition = """
+            -> ProcessScanData
+            ---
+            processed_scan_data_part : int unsigned
+            """
+            
+            
+        def make(self, key):
+            scan_data_dict = (ScanData & key).fetch1()
+            self.insert1(dict(key, processed_scan_data=scan_data_dict['data']))
+            self.ProcessScanDataPart.insert1(
+                dict(key, processed_scan_data_part=scan_data_dict['data'] * 2))
+
+    yield ScanData, ProcessScanData
```

Comment:
Curious if this approach will work:
```suggestion
    yield dict(ScanData=ScanData, ProcessScanData=ProcessScanData)
```

Reply:
That should work

Reply:
Went with this approch